### ISSUE TITLE
Source Documents

### ISSUE THREAD

**Issue by robftg** [robftg/Rocket-Launches#4]
[Wikipedia](https://en.wikipedia.org/wiki/List_of_Space_Shuttle_missions)
[Good One]https://www.space.com/12025-space-shuttle-missions-1981-2011.htmll)

**Comment by frabbitry** [comment]
Just as a quick note, the second one should be https://www.space.com/12025-space-shuttle-missions-1981-2011.html

But good work getting some source documents!

**Comment by robftg** [comment]
https://www.space.com/12064-space-shuttle-missions-1981-1988.html first set of launches

**Comment by frabbitry** [comment]
Yeah I was just noticing that the link was broken because there were two Ls at the end of .html
